docker/Makefile

```diff
@@ -26,13 +25,13 @@ get_gt4py:
 	if [ ! -f ${GT4PY_DIR}/pyproject.toml ]  || \
 	[ "$(shell cd ${GT4PY_DIR} && git describe --tags )" != "${GT4PY_VERSION}"  ]; then \
                 rm -rf ${GT4PY_DIR} && \
-	  	git clone https://github.com/VulcanClimateModeling/gt4py.git ${GT4PY_DIR} && \
+	  	git clone https://github.com/ai2cm/gt4py.git ${GT4PY_DIR} && \
 	  	cd ${GT4PY_DIR} && git checkout ${GT4PY_VERSION};\
 	fi
 
  ## build production container image
 fv3gfs_image: get_gt4py
-	if [ $(PULL) == True ]; then \
+	if [ $(PULL) = True ]; then \
```

Comment:
This one was super weird. It should be `==`, and it works on my machine locally. It is also how `fv3core` has it, and it works on jenkins. However, I'm getting this on jenkins gce: [/bin/sh: 1: [: True: unexpected operator](https://jenkins.ginko.ch/view/pace/job/pace-physics-regression_PR/7/backend=numpy,experiment=c12_6ranks_baroclinic_dycore_microphysics,runaction=run_physics_regression_tests,slave=gce-cpu/consoleFull). But somehow `=` worked [fine](https://jenkins.ginko.ch/view/pace/job/pace-physics-regression_PR/10/backend=numpy,experiment=c12_6ranks_baroclinic_dycore_microphysics,runaction=run_physics_regression_tests,slave=gce-cpu/consoleFull).

Reply:
My guess is it is because PULL is unset on gce. You can fix this by setting the default to `PULL ?= False` instead of removing the definition of PULL above altogether.